lib/src/pubspec.dart

```diff
@@ -189,12 +174,12 @@ class Pubspec extends PubspecBase {
   ///
   /// If [dartSdkWasOverridden] is `false`, this will be identical to
   /// `sdkConstraints["dart"]`.
-  VersionConstraint get originalDartSdkConstraint {
+  VersionConstraint? get originalDartSdkConstraint {
```

Comment:
As I read the code this can never be `null`. Also you do ! to it somewhere, so that would crash us.

Reply:
`late final`